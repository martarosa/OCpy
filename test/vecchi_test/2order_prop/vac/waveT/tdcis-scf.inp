&general 
n_ci=15 
n_ci_read=15  
mol_cc= 0. 0. 0.
medium='vac'
n_f=1
dt=0.20
n_step=10
n_out=1
/ 

&field 
Ffld='snd'
omega(1)=0.26
sigma(1)=10,
t_mid=5000
radiative='non'
npulse=1
fmax(1,1)= 0. 
fmax(1,2)=0. 
fmax(1,3)=0.      
/

&sse
dissipative='non'
/

&spectra 
start=80 
tau= 2000. 2000.
dir_ft= 1. 1. 1. 
/

&pop_coh
tar = 'all'
all_pop= 'yes'
all_coh='yes'
write_bin='n'
/


! Here starts the tdplas part
!
&PROPAGATE  
propagation_type='ief'
interaction_type='pcm'
interaction_stride=1
interaction_init='SCF_ES'
scf_threshold=14
debug_type='non' 
local_field='non'
/

&MEDIUM
medium_type='vac'
medium_init='vac'
bem_type='diag'
bem_read_write='rea'
!print_lf_matrix='non'
/

&SURFACE
input_surface='fil'
/

&EPS_FUNCTION
epsilon_omega='drl'
eps_0=1000.
eps_d=1. 
eps_A=0.110224   
eps_gm=0.000757576
eps_w0=0.  
f_vel=0. 
/

