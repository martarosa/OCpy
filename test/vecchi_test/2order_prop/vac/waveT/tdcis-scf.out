 
 ****************************************************
 ****************************************************
 **                                                **
 **                    WaveT                       **
 **     Evolution of molecular wavefunctions       **
 **   under external electromagnetic perturbations **
 **                                                **
 **                     by                         **
 **                Stefano Corni                   **
 **                Silvio Pipolo                   **
 **               Emanuele Coccia                  **
 **                 Gabriel Gil                    **
 **                Jacopo Fregoni                  **
 **                                                **
 ****************************************************
 ****************************************************
 
 Number to append to dat file           1
 Number of CIS states to be read          15
 Number of CIS states to be used          15
 Molecular center of charge
   0.0000000000000000        0.0000000000000000        0.0000000000000000     
 
 Time step (in au), number of steps, stride  0.20000000000000001               10           1
 
 No external medium, vacuum calculation
 No restart
 Only |k> -> |0> relaxation
 Energy term is propagated analytically
 Matmul is used in the propagation.
 
 Time shape of the perturbing fieldsnd       
 time at the center of the pulse (au):   5000.0000000000000     
 Width of the pulse (time au):   10.000000000000000     
 Frequency (au):  0.26000000000000001     
 Maximum E field (au)   0.0000000000000000        0.0000000000000000        0.0000000000000000     
 Maximum E field (V/m)   0.0000000000000000        0.0000000000000000        0.0000000000000000     
 Maximum intensity (W/cm^2)   0.0000000000000000        0.0000000000000000        0.0000000000000000     
 
 No dissipation
 SSE quantum jumps not written
 
 Excitation energies read from input, in Hartree
           1  0.26953406310081479     
           2  0.28272706166282296     
           3  0.28272706166282296     
           4  0.32398917527124288     
           5  0.32400754992105069     
           6  0.33991264679469169     
           7  0.33992367158457637     
           8  0.35102563499845563     
           9  0.37773135102912786     
          10  0.38570227411575614     
          11  0.38571329890564082     
          12  0.39391206764988601     
          13  0.39486754943989216     
          14  0.39486754943989216     
          15  0.40811934688128526     

 Starting point for FT calculation   80.000000000000000     
 Artificial damping   2000.0000000000000     
 Direction along which the field is oriented   1.0000000000000000        1.0000000000000000        1.0000000000000000     
 
 *************OMP*****************

 OMP parallelization on
The number of processors available =        4
The number of threads available    =        4

 *************OMP*****************
Done reading input & setting up the field, took     0.006s
Done , total elapsed time     0.012s
